### DISCUSSION TITLE
The table of contents is placed on the side to show the reading progress

### DISCUSSION THREAD

**Discussion by xyming108** [adityatelange/hugo-PaperMod#650] [General]
Do you plan to put the directory on the side of the next version to show the reading progress? When I read it now, I have to go to the top to find the directory. I feel it's more troublesome？

**Comment by nielsbrakel** [comment]
Created an PR: https://github.com/adityatelange/hugo-PaperMod/pull/675

**Reply by xyming108** [reply]
thank you，i made it by myself，welcome to my blog：https://www.sulvblog.cn
but i think your design is mush better，i'll use it 😂

**Reply by xyming108** [reply]
I'm sorry to bother you again, but I found that because I use Chinese, as long as the directory is Chinese, the href of the directory will appear Chinese garbled code, which is similar to: href = "#% E4% B8% 801958-1976% E5% B7% a5% E4% B8% 9A% E5% 8C% 96% E5% 88% 9D% E6% 9C% 9F% E7% 9A% 843% E6% AC% A1% E5% 8D% B1% E6% 9C% Ba", resulting in active failure, but the English href can be used normally, I don't know how to solve this problem. I try to use escape or encodeURL or decodeURL, but it doesn't work 😅

**Reply by xyming108** [reply]
![image](https://user-images.githubusercontent.com/51042176/146473561-1b1a031b-f9a6-4367-83a7-432f7c0dbe9a.png)

**Reply by xyming108** [reply]
<div align="center" style="zoom: 10%;">

<span> ![image](https://user-images.githubusercontent.com/51042176/146478325-ccbe1d62-f3ab-4e6c-ba26-f88ea115df94.png) </span>

<span> ![image](https://user-images.githubusercontent.com/51042176/146479264-ed6aa07c-0ab6-4305-aecd-626bdd5985aa.png) </span>


</div>
the active is doesn't work 🤣, because it is not english，

this is url: https://www.sulvblog.cn/posts/tech/hugo_seo

**Reply by nielsbrakel** [reply]
Good find, I will have a look. Will update when I have found a solution.

**Reply by nielsbrakel** [reply]
I pushed an new commit that hopefully fixes all the issues. Could you test it for me? https://github.com/adityatelange/hugo-PaperMod/pull/675/commits/2e2670c3f408fc8ea8b337242a193f4c62713767

**Reply by xyming108** [reply]
thank you, i test it right now

**Reply by xyming108** [reply]
i test it ,i is work, I have deployed to the server，This is the display：
https://www.sulvblog.cn/posts/tech/hugo_seo/
thank you so much for the issues, 👍🏻👍🏻
